src/lib.rs

```diff
@@ -30,22 +31,41 @@ pub fn load_happs_yaml(path: impl AsRef<Path>) -> Result<Vec<Happ>> {
 }
 
 pub async fn install_happs(happ_list: &[Happ], config: &Config) -> Result<()> {
-    let admin_websocket = AdminWebsocket::connect(config.admin_port)
+    let mut admin_websocket = AdminWebsocket::connect(config.admin_port)
```

Comment:
This is probably where retry logic should be located. Something like 10x every 2s?

Reply:
Retry is in #44, please see that

Reply:
Sorry, haven't noticed that!